plugins/Toolbox/src/PackagesModel.py

```diff
@@ -43,13 +43,14 @@ def _update(self):
         items = []
 
         for package in self._metadata:
+            print(package["author"])
             items.append({
                 "id":                package["package_id"],
                 "type":              package["package_type"],
                 "name":              package["display_name"],
                 "version":           package["package_version"],
-                "author_id":         package["author"]["author_id"],
-                "author_name":       package["author"]["display_name"],
+                "author_id":         package["author"]["author_id"] if "author_id" in package["author"] else package["author"]["name"],
```

Comment:
Does the filed "name" in "author" exist? Otherwise it will crash.

Reply:
This will be changed with an update I have made to the packages repo which is awaiting merge. The logic here got a little convoluted because plugins used an `author_name` field while the packages API used: `author.display_name`